### PR TITLE
Create mobilefirst

### PR DESCRIPTION
**Pull request by Diana-Duya**
My second Microverse Project and also my first Mobile version.

### CODE REVIEW

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Pull Request :red_circle: :traffic_light: :rotating_light:

## Hey @Diana-Duya :wave: 

**You have made a great effort on your project :blush: and I can see that but unfortunately I can not proceed with a code review of your pull request due to the following reasons :receipt:**
# 1
### Your Summary
your summary


### Descriptive Summary Example :heavy_check_mark:
Screenshot from 2022-08-19 15-23-49


- [ ] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking:

## 2.
- [ ] Your linters are not working. Please fix them because they prevent you in producing those errors which can break your application :+1:

### Once you implement all the changes you should be ready to submit your project for a review :rocket: :rocket:

Cheers and Happy coding!👏👏👏

### Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. You can reach out to me on slack by searching my name (Hammas Bin Saleem) for any explanation.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            <h2>I’am a Software Developer</h2>
+        </div>
+        <div class="details">
+            <p>I can help you build a product, feature
+               website Look through some of my work
+               and experience! if you like what you see
+               and have a project you need coded
+               don't hesitate to contact me.
+            </p>
+        </div>
```
Comment (Danieldotcomcoder) [inline index.html:42]: Your Design is not showing well at the mobile size width (375px) 
For example: the paragraph is disappearing and there is a big white space at the bottom and your font size looks too large
image
image
although you have all the elements you need them to look good on a screen size of 375px

Kindly fix these issue 👍 😄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+<body>
+    <header>
+        <nav>
+        <!-- navigation bar begins here -->
+            <a href="#"><h2 class="logo">Diana</h2></a>
+            <div class="maimenu">
+                <a href=""><img src="image/Union.svg" alt="union-icon"></a>
+            </div>
+    </nav>
+</header>
```
Comment (Danieldotcomcoder) [inline index.html:23]: Kindly move the logo to the left and and the menu bar to the right a little bit so you can match the given design better 😄 👍 
image

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @Diana-Duya , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+<main>
+    <section id="flex">
+        <!--navigation bar ends  -->
+      <div class="main-section">
+            <div class="first-heading">
+              <h2>Hey There.
+                  I'm Diana</h2>
+        </div>
+        <div class="secon-heading">
+            <h2>I’am a Software Developer</h2>
```
Comment (franclobo) [inline index.html:33]: - [ ] **[OPTIONAL]** We could use `<h1>` tag for the title, to follow the best practices of HTML & CSS. 😃

File: index.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            </p>
+        </div>
+        <div class="social-media">
+            <ul>
+                <li><a href=""><img src="image/github.svg" alt="github-logo"></a></li>
+                <li><a href=""><img src="image/linked-in.svg" alt="linked-in-logo"></a></li>
+                <li><a href=""><img src="image/medium.svg" alt="medium-logo"></a></li>
+                <li><a href=""><img src="image/twitter.svg" alt="twitter-logo"></a></li>
+                <li><a href=""><img src="image/Vector3.svg" alt="Vector (3)-logo"></a></li>
+            </ul>
```
Comment (franclobo) [inline index.html]: - [ ] **[OPTIONAL]** Kindly order the icons to be similar to the Figma template. 👍🏽 

- Figma template

image

- Your design

image

File: style.css [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  color: #fff;
+  padding-top: 20px;
+}
+
+.social-media ul {
+  display: flex;
+  gap: 1rem;
+  padding: 0 20px;
+  padding-top: 15px;
+}
```
Comment (franclobo) [inline style.css:86]: - [ ] **[OPTIONAL]** Please align the icons in the horizontal axis, for a better user experience. ✌🏽 

> You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- Figma template

image

- Your design.

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Diana-Duya , 

You are doing well, keep it on!! 😃 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- Descriptive Readme File ✔
- No linter errors ✔
- Correct Github flow ✔


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] **[OPTIONAL]** Kindly, we could avoid sections on Readme file that we are not using.

image

- [ ] **[OPTIONAL]** Please, we could add a title and list our actions on the correspondat feature brach to have a descriptive Pull Request.

image



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by phelian23 — APPROVED** [review]
Hi @Diana-Duya,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**